convert.py

```diff
@@ -1033,7 +1033,7 @@ def add_meta_arch(self, params: Params) -> None:
         if params.n_ctx == 4096:
             name = "LLaMA v2"
         elif params.path_model is not None:
-            name = str(params.path_model.parent).split("/")[-1]
+            name = params.path_model.name
```

Comment:
Isn't this code supposed to get the basename of the containing folder? Maybe `params.path_model.parent.name`.